tutorials/dataframe/df103_NanoAODHiggsAnalysis.py

```diff
@@ -206,23 +206,23 @@ def plot(sig, bkg, data, x_label, filename):
     h_cmb.GetXaxis().SetTitleSize(0.04)
     h_cmb.GetYaxis().SetTitle("N_{Events}")
     h_cmb.GetYaxis().SetTitleSize(0.04)
-    h_cmb.SetLineColor(ROOT.kRed)
+    h_cmb.SetLineColor("kRed")
     h_cmb.SetLineWidth(2)
     h_cmb.SetMaximum(18)
     h_cmb.SetStats(False)
 
     h_bkg.SetLineWidth(2)
     h_bkg.SetFillStyle(1001)
-    h_bkg.SetLineColor(ROOT.kBlack)
-    h_bkg.SetFillColor(ROOT.kAzure - 9)
+    h_bkg.SetLineColor("kBlack")
```

Comment:
What is the typical Python behavior if the name is incorrect spelt?  I.e. should the new `SetLineColor` (and co) print an error message?

Reply:
Right now, it will just silently do nothing. Not sure yet if this should be treated as an Error: also when you call the old `SetLineColor()` with an invalid integer, it doesn't complain, and when drawing it falls back to black.

Reply:
You are right, the cases are not comparable. Would raising a `std::invalid_argument` be okay for you?

It would look like this:
```txt
Traceback (most recent call last):
  File "/home/rembserj/spaces/master/root/src/root/test.py", line 5, in <module>
    h.SetLineColor("kReed")
cppyy.gbl.std.invalid_argument: void TAttLine::SetLineColor(const string& lcolor) =>
    invalid_argument: "kReed" is not a valid color name
```

Reply:
Sounds fair.